src/settings.py

```diff
@@ -245,6 +245,9 @@ def get_arg_specific_settings(args: Dict[str, Any]) -> Dict[str, Any]:
                     # lower level, so tasks take more actions to complete.
                     "behavior": 1000,
                     "pybullet_blocks": 1000,
+                    # For the very simple TouchPoint environment, restrict
+                    # the horizon to be shorter.
+                    "touch_point": 15,
```

Comment:
is this the horizon at the action level or the option level?

Reply:
action level